From 631c4c7b57551fabf4799ef36e41f6d6e651761e Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:34:50 -0400
Subject: [alsa] hda/realtek: Add support of ALC667 codec

Message-id: <140972823573-331-patchset-jkysela@redhat.com>
Patchwork-id: 91027
O-Subject: [RHEL 7.1 PATCH 330/442] ALSA: hda/realtek - Add support of ALC667 codec
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

New codec suooprt of ALC667.

Signed-off-by: Kailang Yang <kailang@realtek.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 72009433b28c2bf27f69e3e28f6cb425cefb76c0)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index ce9bda8..d77745a 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -6274,6 +6274,7 @@ static const struct hda_codec_preset snd_hda_preset_realtek[] = {
    .patch = patch_alc662 },
  { .id = 0x10ec0663, .name = "ALC663", .patch = patch_alc662 },
  { .id = 0x10ec0665, .name = "ALC665", .patch = patch_alc662 },
+ { .id = 0x10ec0667, .name = "ALC667", .patch = patch_alc662 },
  { .id = 0x10ec0668, .name = "ALC668", .patch = patch_alc662 },
  { .id = 0x10ec0670, .name = "ALC670", .patch = patch_alc662 },
  { .id = 0x10ec0671, .name = "ALC671", .patch = patch_alc662 },
-- 
1.7.1